modules/data-entry/src/main/frontend/src/questionnaire/Questionnaire.jsx

```diff
@@ -126,6 +132,31 @@ let Questionnaire = (props) => {
     });
   }, []);
 
+  let dropdownList = (
+      <List>
+        <ListItem className={classes.actionsMenuItem}>
+          <ExportButton
+              entityData={data}
+              entryPath={data ? data["@path"] : `/Questionnaires/${id}`}
+	          entryName={questionnaireTitle}
+	          entryType="Questionnaire"
```

Comment:
We're not actually exporting the questionnaire, but the forms associated with it.
```suggestion
	          entryType="forms"
```